osf/models/user.py

```diff
@@ -369,6 +369,8 @@ class OSFUser(DirtyFieldsMixin, GuidMixin, BaseModel, AbstractBaseUser, Permissi
     # The time at which the user agreed to our updated ToS and Privacy Policy (GDPR, 25 May 2018)
     accepted_terms_of_service = NonNaiveDateTimeField(null=True, blank=True)
 
+    chronos_user_id = models.TextField(null=False, blank=True, db_index=True)
```

Comment:
should null be false or should we make it True?

Reply:
`True` would make for a faster migration.